application.go

```diff
@@ -191,13 +191,13 @@ func (r *Application) HasHealthChecks() bool {
 }
 
 // Deployments retrieves the application deployments ID
```

Comment:
The comment won't pass `golint` now. Can you change it please to `DeploymentIDs does something`?